content/blog/2021-04-22-girls-day/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title:  "Girls Day 2021"
+date:   2021-04-22T13:00:00+0100
+categories: [berichte]
+tags: [workshop, einstieg, spiel, grundschule, programmieren, scratch]
+authors: [Bruno]
+---
+Bei unserem Workshop zum Girls Day 2021 haben wir vor allem das Zusammenspiel zwischen Software und Gesellschaft erforscht.
+Am 22. April 2021 haben sich 18 Schülerinnen zusammen mit 7 Mentorinnen\* zusammen gesetzt (virtuell natürlich) und sind
```

Comment:
Das Sternchen am Ende wird gemacht wenn man sagen will "Alle Frauen, und damit meinen wir alle Personen die sich als Frau fühlen". Wenn du es in der Mitte setzt meint es alle Geschlechter.

Reply:
Ah ok. Das wusste ich noch nicht. Hatte mit Luis geschrieben und der meinte, dass der Doppelpunkt eventuell besser ist, weil der von Screenreadern wohl besser gelesen wird. Wollten das heute Abend noch diskutieren. Würde den nächsten Pull-Request dann danach machen.

Reply:
ich hab da keine präferenz, wir können gern zum : switchen! ich denke es ist ok wenn du das mit luis abgesprochen hast, wir sollten dann nur eine trello karte machen, damit das überall anders auch angepasst wird